google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -338,6 +338,10 @@ clouddebug.runconfig.documentation.url.text=click here
 
 gcs.panel.toolbar.refresh.hover.text=Refresh
 gcs.panel.toolbar.refresh.hover.description=Refresh Google Cloud Storage buckets
+gcs.panel.bucket.listing.no.project.selected=To view your buckets select a Cloud Project.
```

Comment:
I'm not totally sure about this one. We've been using Cloud Project in other cases. I think if we think of it as a single entity then its correct. I'll leave for now unless we can find evidence (like in the docs) that shows otherwise.